03_letters/pfannerj/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	mainout = &buf
+	main()
+	expected := `a:2
+b:1`
+	actual := buf.String()
+	if expected != actual {
+		t.Errorf(expected, actual, "Unexpected output in main()")
+	}
+}
+
+var lettersCases = map[string]struct {
+	input    string
+	expected map[rune]int
+}{
+	"Standard": {input: "aba", expected: map[rune]int{97: 2, 98: 1}},
+	"Same":     {input: "cccccccccccccccccccc", expected: map[rune]int{99: 20}},
+	"Empty":    {input: "", expected: map[rune]int{}},
+	"Return":   {input: "ab\na\n", expected: map[rune]int{10: 2, 97: 2, 98: 1}},
+	"Slash":    {input: "ab/na/n", expected: map[rune]int{47: 2, 97: 2, 98: 1, 110: 2}},
+	"LongMixed": {input: "zabaaarzzzASYbcd@&^!*JHDJKHHGFGD^!*Dar",
+		expected: map[rune]int{33: 2, 38: 1, 42: 2, 64: 1, 65: 1, 68: 3, 70: 1, 71: 2, 72: 3, 74: 2, 75: 1, 83: 1,
```

Comment:
not too fussed about but this lines are very long, I tend to stick to 80, but it is a very minor issue, the only I can see, albeit very minor

Reply:
Thanks!  Yeah, I got an error from the lint and split the lines so that it was happy but there still could be a better way to format the long lines.. will investigate but leaving it as-is for now...